Fix: add guard for dev mode (#165)
ghostdevv/jellycommands: docs/guide/commands/dev.md

For additional options, see [dev options](/api/client#dev).

:::tip NOTE

> If dev mode is enabled in any way, atleast one guild ID must be specified within [`dev.guilds`](/api/client#dev)

Reviewer: ```suggestion
If `dev` mode is enabled, at least one guild id must be specified within [`dev.guilds`](/api/client#dev)
```
